Runtime/Resources/UIWidgets_canvas_strokeAlpha.shader

```diff
@@ -0,0 +1,65 @@
+Shader "UIWidgets/canvas_strokeAlpha"
+{
+    Properties {
+       _SrcBlend("_SrcBlend", Int) = 1 // One
+       _DstBlend("_DstBlend", Int) = 10 // OneMinusSrcAlpha
+       _StencilComp("_StencilComp", Float) = 3 // - Equal, 8 - Always 
+    }
+   
+    SubShader {
+        ZTest Always
+        ZWrite Off
+        Blend One OneMinusSrcAlpha
+        
+        Stencil {
+            Ref 128
+            Comp [_StencilComp]
+            Pass IncrSat
+        }
+
+        Pass { // 0, color
+            CGPROGRAM
+            #define UIWIDGETS_COLOR
+            #include "UIWidgets_canvas.cginc"
+            #pragma vertex vert
+            #pragma fragment frag_stroke_alpha
+            ENDCG
+        }
+        
+        Pass { // 1, linear
```

Comment:
please make sure that all these Passes work correctly, especially rarely used ones, e.g., Pass 1 - Pass 3